examples/03_special_classes/04_Object.md

```diff
@@ -26,24 +25,24 @@ fun main(){
 
 </div>
 
-1. blueprint definition
-2. method definition
-3. instance creation
-4. method calls 
+1. Defining a blueprint.
+2. Defining a method.
+3. Creating instances.
+4. Calling the method on instances. 
 
-Easy: we create two objects, both *instances* of LuckDispatcher class.
+In Kotlin you also have also the [**object** keyword](https://kotlinlang.org/docs/reference/object-declarations.html). It is used to obtain a *data type with a single implementation*.
```

Comment:
In Kotlin you also have the [**object** keyword](https://kotlinlang.org/docs/reference/object-declarations.html). It is used to obtain a *data type with a single implementation*.

2 also == too many also